.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -220,3 +222,21 @@ def skip_if_no_helm_changes():
         return None
 
     return "No helm changes"
+
+
+@functools.lru_cache(maxsize=None)
+def skip_if_no_docs_changes():
+    if not is_feature_branch(os.getenv("BUILDKITE_BRANCH")):
+        return None
+
+    # If anything changes in the docs directory
+    if any(Path("docs") in path.parents for path in get_changed_files()):
+        logging.info("Run docs steps because files in the docs directory changed")
+        return None
+
+    # If anything changes that uses the literalinclude docstring
+    if any("literalinclude" in path.read_text() for path in get_changed_files()):
+        logging.info("Run docs steps because files with the literalinclude docstring changed")
+        return None
```

Comment:
im not sure this is right - the docs refer to python files using pointers and markers 
https://github.com/dagster-io/dagster/blame/master/examples/docs_snippets/docs_snippets/concepts/assets/materialization_io_managers.py#L38

and i think the docs tests need to re-run if the inlined snippets change 

defer to @yuhan on the best strategy here - i think most snippets are in `docs_examples` directory

Reply:
I took the guidance from here:

https://github.com/dagster-io/dagster/blob/398ea993c5f443f606e6fb447eda6f631f4bc9b1/.buildkite/dagster-buildkite/dagster_buildkite/steps/docs.py#L15-L21

Maybe that's outdated?

I don't see a the `examples/docs_snippets` though. We ran it on changes to `examples` previously but I didn't realize we have that snippets directory hidden amongst the actual examples.

Reply:
So the `literalinclude` no longer drives any functionality?

Reply:
Also if "most" are in `examples/docs_snippets`, where are the rest? Or are all in there?

Reply:
alex's right. most are in examples/docs_snippets, the rest are in examples but outside the docs snippets (those are mostly standalone guides).

right we no longer use `literalinclude`.